doc/source/release-notes/Percona-XtraDB-Cluster-5.6.44-28.34.rst

```diff
@@ -0,0 +1,38 @@
+.. rn:: 5.6.44-28.34
+
+================================================================================
+|PXC| |release|
+================================================================================
+
+Percona is glad to announce the release of |PXC| |release|
+on |date|.  Binaries are available from the `downloads section
+<http://www.percona.com/downloads/Percona-XtraDB-Cluster-56/>`_ or from our
+:ref:`software repositories <installation>`.
+
+|PXC| |release| is now the current release, based on the following:
+
+- `Percona Server 5.6.44
+  <https://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.44-86.0.html>`_
+- Codership WSREP API release 5.6.43
+- Codership Galera library 3.25
+
+All Percona software is open-source and free.
+
+Bugs Fixed
+================================================================================
+
+- :jirabug:`2434`: The password could not be visible in a conflict
+  debug message.  The error message was changed to hide the password.
+- :jirabug:`2480`: In some cases, |PXC| could not replicate
```

Comment:
We do not allow the use of USER() or CURRENT_USER() in any ALTER statement, not just ALTER TABLE (for instance ALTER USER CURRENT_USER() is also blocked).